src/Symfony/Component/Uid/AbstractUid.php

```diff
@@ -127,6 +127,14 @@ public function toRfc4122(): string
         return substr_replace($uuid, '-', 23, 0);
     }
 
+    /**
+     * Returns the identifier as a prefixed hexadecimal case insensitive string.
+     */
+    public function toHex(): string
+    {
+        return '0x'.bin2hex($this->toBinary());
```

Comment:
Do we really need the `0x` prefix? Shouldn't it be added by the outside/consumer?

Reply:
The whole idea is the use of hex value with `0x` prefix for fast uses where data stored as binary e.g. SQL